lib/python/mod_python/psp.py

```diff
@@ -271,7 +271,7 @@ def __init__(self, label, file, cache):
                     psp.error_page.run({"exception": (et, ev, etb)}, flush)
                 else:
                     if PY2:
-                        raise et, ev, etb
+                        exec('raise et, ev, etb')
```

Comment:
I see now, I guess it is explained here: https://wiki.python.org/moin/PortingToPy3k/BilingualQuickRef#raise 

Looks good then, I'm going to merge this, thanks for the fix!